modules/nw-route-specific-annotations.adoc

```diff
@@ -7,6 +7,10 @@
 
 The Ingress Controller can set the default options for all the routes it exposes. An individual route can override some of these defaults by providing specific configurations in its annotations.
 
+[IMPORTANT]
+====
+To create a whitelist with multiple source IPs or subnets, use a space-delimited list. Any other delimiter type causes the list to be ignored without a warning or error message.
```

Comment:
One question related to the conscious language project: 
could we change "whitelist" to "allowlist"?

Reply:
Nevermind on this question. Looks like there are 21 instances in the file that we'd need to address.